src/utils/validate.js

```diff
@@ -0,0 +1,40 @@
+const errors = require('../errors');
+
+const Validate = {
+  requiredFields(fieldsArr) {
+    this.validate = function(data) {
```

Comment:
curious on why you decided to use this pattern instead of putting the input data as second parameter on `requiredFields` function instead?

Reply:
Oh this is so that I can get a reusable object. i.e.
```js
const data1 = {};
const data2 = { someField: 'not missing', someOther: 'not missing' };
const checker = Validate.requiredFields(['someField', 'someOther']);
checker.validate(data1); # fail 
checker.validate(data2); # success
```